tests/expected/intrinsics/copy-nonoverlapping/copy-unaligned-dst/expected

```diff
@@ -0,0 +1,2 @@
+FAILURE\
+`dst` is properly aligned
```

Comment:
The failure message usually describes an error or an expectation that should hold.

Reply:
The expectation is that the pointers are aligned. We can change all alignment checks to "X must be properly aligned", would that do it for you?

Reply:
Yes. This message I read as there is a failure because `dst` is properly aligned. Thanks!